docs/user_guide/layout.rst

```diff
@@ -589,3 +590,15 @@ If you do specify ``html_last_updated_fmt`` but don't include the :code:`last-up
 
 The tag's ``content`` attribute will follow the format specified in the ``html_last_updated_fmt`` configuration variable.
 
+
+.. _references:
+
+References
```

Comment:
I believe this is unnecessary as section titles are automatically made into crossref targets? But it doesn't hurt anything either and I sorta prefer the explicitness.